apps/E2E/src/common/BasePage.ts

```diff
@@ -363,15 +363,15 @@ export abstract class BasePage {
   // Returns: String
   // Returns the identifier of the primary UI element used for testing on the given test page.
   get _primaryComponentName(): string {
-    console.error('Each class extending BasePage must implement its own _primaryComponentName method.');
+    console.warn('Each class extending BasePage must implement its own _primaryComponentName method.');
```

Comment:
Change warning message:

"Please verify whether or not your page object should implement _primaryComponentName"